src/test/java/org/mockito/internal/runners/util/RunnerProviderTest.java

```diff
@@ -17,7 +17,7 @@ public void shouldCreateRunnerInstance() throws Throwable {
         //given
         RunnerProvider provider = new RunnerProvider();
         //when
-        RunnerImpl runner = provider.newInstance("org.mockito.internal.runners.SilentJUnitRunner", this.getClass(), null);
+        InternalRunner runner = provider.newInstance("org.mockito.internal.runners.DefaultInternalRunner", this.getClass(), null);
```

Comment:
OK, about the JUnit on the classpath, but will the `RunnerFactory` be used at all then ? The only reasonable use is in the `Silent` runner class which itself extends the JUnit `Runner` class.